hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java

```diff
@@ -76,17 +78,16 @@ public static String getTestMethodName() {
     }
 
     /**
-     * Creates a BlockJUnit4ClassRunner to run {@code klass}
+     * Creates a BlockJUnit4ClassRunner to run {@code classType}
      *
      * @throws org.junit.runners.model.InitializationError if the test class is malformed.
      */
-    public AbstractHazelcastClassRunner(Class<?> klass) throws InitializationError {
-        super(klass);
```

Comment:
With type of class you start to enter complex type systems. Where a type of a type (class) is also a 'kind'. So it puts me on the wrong foot.